lib/Cmts.ml

```diff
@@ -564,10 +564,10 @@ module Unwrapped = struct
     | _ -> wrap "(*" "*)" @@ str s
 end
 
-let fmt_cmt (cmt : Cmt.t) ~wrap:wrap_comments ~ocp_indent_compat ~fmt_code
-    pos =
+let fmt_cmt (cmt : Cmt.t) ~style ~ocp_indent_compat ~fmt_code pos =
   let mode =
     match cmt.txt with
+    | txt when Poly.(style = `Verbatim) -> `Verbatim ("(*" ^ txt ^ "*)")
```

Comment:
Slightly unrelated, the `ocp_indent_compat` branch of `Unwrapped.fmt` looks a lot like `` `Verbatim ``. Perhaps it can be moved to this function (there's an extra break depending on `pos`) and the `ocp_indent_compat` flag checked here.

Reply:
Indeed I remember that we needed something similar for the Jane Street codebase.
@ceastlund in your experiments could you try to use my branch `verbatim-comments-janestreet` and see if it improves the results? (in this branch I'm not relying on ocp_indent_compat, but instead on comments=vrebatim to get the same behaviorm and the janestreet profile sets comments=verbatim)

Reply:
This looks like what I had in mind.